src/pages/Vaults/index.tsx

```diff
@@ -35,15 +37,29 @@ const VaultOverview = (): JSX.Element => {
           {vaultOverview?.vaults?.map((vault) => (
             <GridItem key={vault.collateralId} mobile={{ span: 4 }} desktop={{ span: 3 }}>
               <VaultCard
-                collateralSymbol={getCurrencySymbol(vault.collateralId)}
-                wrappedSymbol={getCurrencySymbol(vault.wrappedId)}
+                collateralSymbol={vault.collateralId}
+                wrappedSymbol={vault.wrappedId}
                 pendingRequests={vault.pendingRequests}
                 apy={safeRoundTwoDecimals(vault.apy.toString())}
                 collateralScore={safeRoundTwoDecimals(vault.collateralization?.mul(100).toString(), '∞')}
                 link={`${accountAddress}/${vault.collateralId}/${vault.wrappedId}`}
               />
             </GridItem>
           ))}
+          <GridItem mobile={{ span: 4, start: 1 }} desktop={{ span: 12, start: 1 }}>
+            <NewVaultsTable
+              data={availableVaults.map((vault) => ({
+                collateralCurrency: vault.collateralCurrency,
+                wrappedCurrency: vault.wrappedCurrency,
```

Comment:
On L40 we use `collateralSymbol` and on L52 we use `collateralCurrency` to denote the same property, I think we should unify it since both components are part of the component library. I think we use 3 different words: `symbol`, `id`, and `currency` for the same use case now in different parts of the codebase which might be little confusing. What are your thoughts on this? also cc @crypto-engineer

Reply:
> I think for the UI components we should probably restrict our use to symbol, as that's always what we would want to render. I preferred having all uppercase in the string literal, but I could live with `kBTC` if it made things a bit more understandable.

I may be wrong, but I think the currency symbols are always in upper case, so using `KBTC` is probably correct.

Reply:
I'll check